versions.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
```

Comment:
I'd suggest to keep the name of the file to version.tf (without s) for overall consistency across the DAs / modules

Reply:
done